webapp/custom-webpack.config.js

```diff
@@ -15,6 +16,12 @@ module.exports = {
       '../../js/dropdown.jquery': 'node_modules/bootstrap/js/dropdown',
       'bikram-sambat': 'node_modules/bikram-sambat',
       'messageformat': 'node_modules/messageformat/index',
+      'lodash/core': 'node_modules/lodash/core',
+      'lodash/uniqBy': 'node_modules/lodash/uniqBy',
+      'lodash/flatten': 'node_modules/lodash/flatten',
+      'lodash/intersection': 'node_modules/lodash/intersection',
+      'lodash/partial': 'node_modules/lodash/partial',
+      'lodash/uniq': 'node_modules/lodash/uniq',
```

Comment:
1. Why doesn't tree shaking work out of the box?
2. Can we alias these to `lodash-es` to remove the dependency on `lodash`?

Reply:
Lodash-es doesn't have the concept of "core", that we use in shared libraries :( 

Lodash-es is not as closely maintained as lodash, so it'd be best we steer off of it completely. 
Treeshaking is, aparently, fragile business.

Reply:
I can alias everything else to lodash-es, except for core.